ping/ping.c

```diff
@@ -599,6 +636,11 @@ main(int argc, char **argv)
 		error(2, 0, "%s: %s", target, gai_strerror(ret_val));
 
 	for (ai = result; ai; ai = ai->ai_next) {
+		if (rts.opt_verbose)
+			printf("ai->ai_family: %s, ai->ai_canonname: %s\n",
+				   str_family(ai->ai_family),
+				   ai->ai_canonname ? ai->ai_canonname : _("not found"));
+
```

Comment:
@richiejp NOTE: I put here `"not found" in case of `NULL`. The rest of strings should be handled well in `str_family()` and `str_socktype()`.

BTW I cannot use ` ai->ai_canonname ?: _("not found"));` due:

```
../ping/ping.c:642:54: warning: ISO C forbids omitting the middle term of a ‘?:’ expression [-Wpedantic]
  642 |                                    ai->ai_canonname ?: _("not found"));

```
A bit surprising, I suppose it's not by C99 but by other options. Flags configured with meson:
```
 default_options : [
        'c_std=c99',
        'warning_level=3',
        'localstatedir=var',
],
```

Reply:
yeah it's not in any C standard IIRC.

More importantly is "not found" a valid name? It would be nice to have some symbols in there which make it clear. Or print the name in quotes, then just use an empty string as default.

Reply:
@richiejp I put just empty string (`""`), because it'd have to be some deeper error. Normally `getaddrinfo()` returns `-1`, thus executing gets quit by `error(2, 0, "%s: %s", target, gai_strerror(ret_val));`:

https://github.com/iputils/iputils/blob/43e38f2215a16e8a2c245e3176bea2adb3ede958/ping/ping.c#L597-L599